cmd/cluster-agent/subcommands/start/command.go

```diff
@@ -106,10 +107,16 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				collectors.GetCatalog(),
 				fx.Supply(workloadmeta.Params{
 					InitHelper: common.GetWorkloadmetaInit(),
-					AgentType:  workloadmeta.ClusterAgent,
```

Comment:
Do we need to remove the `AgentType` here ?

Reply:
Good catch! It must be a typo.